cluster-autoscaler/FAQ.md

```diff
@@ -304,6 +305,112 @@ It can be added to (or removed from) a node using kubectl:
 kubectl annotate node <nodename> cluster-autoscaler.kubernetes.io/scale-down-disabled=true
 ```
 
+### How can I configure overprovisioning with Cluster Autoscaler?
+
+Below solution works since version 1.1 (to be shipped with Kubernetes 1.9).
+
+Overprovisioning can be configured by creating deployment which:
+ * reserves resources proportional to the number of cores in the cluster (i.e 200 millicores per core),
+ * frees resources when they are needed to schedule new pods,
+ * forces Cluster Autoscaler to scale up the cluster when there is not enough free resoures.
+
+Deployment described above can be created using [Priority Preemption](https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/)
+and [Horizontal Cluster Proportional Autoscaler](https://github.com/kubernetes-incubator/cluster-proportional-autoscaler)
+
+Configuration:
+
+1. Enable priority preemption in your cluster. It can be done by exporting following env
+variables before executing kube-up (more details [here](https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/)):
+```sh
+export KUBE_RUNTIME_CONFIG=scheduling.k8s.io/v1alpha1=true
+export ENABLE_POD_PRIORITY=true
+```
+
+2. Define priority classes. Priority 0 will be reserved for overprovisioning pods as it is the lowest
```

Comment:
You have to go BELOW 0 with overprovisioning pods. 0 is the default priority and thus should preempt the pause pods.

Reply:
done